spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -111,6 +114,21 @@ public void setTransactionSynchronizationFactory(TransactionSynchronizationFacto
 		this.transactionSynchronizationFactory = transactionSynchronizationFactory;
 	}
 
+	/**
+	 * Return the default error channel if the error handler is explicitly provided and
+	 * it is a {@link MessagePublishingErrorHandler}.
+	 * @return the channel or null.
+	 * @since 4.3
+	 */
+	public MessageChannel getDefaultErrorChannel() {
+		if (!this.errorHandlerIsDefault && this.errorHandler instanceof MessagePublishingErrorHandler) {
+			return ((MessagePublishingErrorHandler) this.errorHandler).getDefaultErrorChannel();
+		}
+		else {
+			return null;
```

Comment:
Yeah. Makes sense.
Finally they even can connect them on their own. Since the default `errorChannel` is always implied.
We just don't enforce an explicit connection to render.
OK. That's fine.
Let's reflect that in the Reference Manual as well: https://jira.spring.io/browse/INT-4008!